src/components/modal/examples/Modal.vue

```diff
@@ -92,7 +92,9 @@ export default {
 },
   data() {
     return {
-      opened: this.$route.name === 'Modals',
+      // TODO not sure why this was setup like this?
+      opened: false,
+      // opened: this.$route.name === 'Modals',
```

Comment:
Side fix:  I found this really difficult to develop with - I needed to see the page state before and after modal load. I wasn't sure if there was a reason we had it load on going to the route, but if not I'd love to leave it like this.